src/App.tsx

```diff
@@ -1,19 +1,48 @@
 import React from 'react';
 import './App.css';
-
+import { TodoList } from './TodoList';
+import todos from './api/todos';
 import users from './api/users';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
+const todosWithUser: Todos[] = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId) || null,
+}));
+
+interface State {
+  todoList: Todos[];
+  userList : Users[];
```

Comment:
Todos - множественное число, получается массив массивов Todos[]

```suggestion
  todoList: Todo[];
  userList : User[];
```